tests/voq/test_voq_fabric_status_all.py

```diff
@@ -0,0 +1,130 @@
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import pytest
+import yaml
+
+logger = logging.getLogger(__name__)
+
+# This test only runs on t2 systems.
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+# This test checks the fabric link status.
+# It loads the reference data for a linecard,
+# and runs the CLI command to get the link status
+# on the system and compares the output.
+
+
+def test_voq_fabric_link_status(duthosts):
+    """Check if the fabric serdes links are all up
+    """
+    logger.info("Checking fabric serdes links")
+
+    # Get hwSku for Fabriccards from the supervisor.
+    fabric_sku = None
+    for duthost in duthosts:
+        if duthost.facts['slot_num'] < 3:
+            fabric_sku = duthost.facts['hwsku']
+            break
+    pytest_assert(fabric_sku, "Need to add hwSku information for sup")
+
+    # Test fabric links status in Linecards, and get the expected link
+    # information for Fabriccards.
+    keys = []
+    # There are 12 asic on Supervisor now.
+    totalAsics = 12
```

Comment:
this assumption may not be valid. All testbeds may not have all the fabric cards and some chassis platforms may have more than 12 fabric asics.

Reply:
this test also need https://github.com/sonic-net/sonic-swss/pull/2652 to be merged to get the fabric counters.